source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -15,7 +15,8 @@ Update Arrays in a Document
 Overview
 --------
 
-In this guide, you can learn how to update array elements in a document.
+In this guide, you can learn how to update array elements in a document
+or multiple documents.
```

Comment:
Minor suggestion (feel free to take or leave this one) but "a document or multiple documents" feels too wordy to me, maybe something like:
```suggestion
In this guide, you can learn how to update array elements in one,
or multiple documents.
```

Reply:
decided to go with "one or more documents"